files/web.xml

```diff
@@ -0,0 +1,4737 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
```

Comment:
@fabricetriboix are there any modifications done from the default web.xml file..?

Reply:
Yes, there are

Reply:
Could we know what are the modifications made? Because taking the reference from Ansible there are no changes made in web.xml( correct me if I am wrong).

Reply:
@naveen-armedia (1) It seems I did commit the correct file, this is fixed now. (2) Here is the diff:

```
--- a/files/web.xml
+++ b/files/web.xml
@@ -4734,4 +4734,17 @@
         <welcome-file>index.jsp</welcome-file>
     </welcome-file-list>
 
+    <error-page>
+        <error-code>404</error-code>
+        <location>/error.jsp</location>
+    </error-page>
+    <error-page>
+        <error-code>403</error-code>
+        <location>/error.jsp</location>
+    </error-page>
+    <error-page>
+        <error-code>500</error-code>
+        <location>/error.jsp</location>
+    </error-page>
+
 </web-app>
```